calculator.rb

```diff
@@ -0,0 +1,55 @@
+# 1st method - evaluate_oper(operator): evaluate user input to see if it is one of the four avaliable operators (either symbol or word) for this calculator program. If yes, method will return the input; if no, method will display error message. Assumption: only lowercase input
+def evaluate_oper(operator)
+  if ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"].include?(operator)
+    return operator
+  else
+    return "This is an invalid input"
+  end
+end
+# 2nd method - evaluate_num(number): evaluate user input to see if it is a number. Prior - all input are strings; Post - all input are converted into float [number will remain float; letter will become 0; expect for integer 0]. Assumption: don't need to handle 0 for this assignment *** TA Kaida helped me on this method
+def evaluate_num(number)
```

Comment:
Similar to evaluate_op, consider using a while loop to continually ask for user input until it is valid.